src/IO/output_LTO_perf.jl

```diff
@@ -0,0 +1,141 @@
+            # println(@sprintf("Running LTO point %.1f %%", TS*100))
+            ac.pared[ieFe, iptest] = Foo*TS
+            TASOPT.tfcalc!(ac.pari, ac.parg, view(ac.parad, :, iptest), 
+                                    view(ac.pared, :, iptest), ac.wing,
+                                    ip, icall, icool, initeng)
+            
+            F_total = ac.pared[ieFe, iptest]# * ac.parg[igneng]
+            mdotf = ac.pared[ieff, iptest] * ac.pared[iemcore]
+            P3_kPa = ac.pared[iept3, iptest]/1000.0
+            T3_K   = ac.pared[ieTt3, iptest]
+            EI = EINOx(ac, iptest; method=method)
+            EIs[i] = EI
+            mfs[i] = mdotf
+        
+            println(fileout, 
+            @sprintf("%2d) %4.2f %15.5f %15.5f %15.5f %15.5f",
+            i, TS, F_total/1000, mdotf, EI, EI*mdotf))
+
+        end
+
+        println(fileout, "------------------------- AEIC ENG_EI output --------------------------")
+        for (i,j) in zip([3,2,1,4], [1,2,3,4])
+            println(fileout,@sprintf("%s, %d, 0.00, 0.00, %10.5f, 0.00, 1.00,  %10.5f, %s, 1.0",
+                                        name, j, EIs[i], mfs[i], name))
+        end
+
+        return EIs, mfs
+end
+
+"""
+    EINOx3(P3_kPa,T3_K, sp_humidity = 0.00634)
+
+
+Returns the EI(NOₓ) for a CFM56 level engine.
+Uses a third order polynomial in T₃ 
+Assumes a default specific humidity of 0.00634 kg water/kg dry air per 
+ICAO Annex 16 Vol. II (part 2.1.4.1)
+"""
+function EINOx3(P3_kPa, T3_K, sp_humidity = 0.00634, ac_type = "1")
+    # Constants derived using a CRN model for a CFM56 tech level engine
+        
+    if ac_type == :wide
+        a = 1.01084407e-07
+        b = -2.12716481e-04
+        c = 1.50618950e-01
+        d = -3.49737491e+01
+    else
+        a = 6.25528852e-08
+        b = -1.17064467e-04
+        c = 7.36953400e-02
+        d = -1.50392850e+01
+    end
+
+    H = -19.0*(sp_humidity - 0.00634)
+    
+    EI_NOx = exp(H)*P3_kPa^0.4 * (a*T3_K^3 + b*T3_K^2 + c*T3_K + d)
+   
+    return EI_NOx
+end
+"""
+    EINOx4(P3_kPa, T3_K, sp_humidity = 0.00634)
+
+Returns the EI(NOₓ) for a CFM56 level engine.
+Uses a fourth order polynomial which behaves a little better at low T₃ than the cubic
+Assumes a default specific humidity of 0.00634 kg water/kg dry air per 
+ICAO Annex 16 Vol. II (part 2.1.4.1)
+"""
+function EINOx4(P3_kPa, T3_K, sp_humidity = 0.00634, ac_type = "1")
+    if ac_type == :wide
+        a = 9.15868718e-10
+        b = -2.60435361e-06
+        c = 2.75257358e-03
+        d = -1.27746983e+00
+        e = 2.19853429e+02
+    else
+        a = 4.85354237e-11
+        b = -6.51089333e-08
+        c =  7.19366066e-06
+        d = 2.06850617e-02
+        e =  -6.69412110e+00
+    end
+
+    H = -19.0*(sp_humidity - 0.00634)
+
+    EI_NOx = exp(H)*P3_kPa^0.4 * (a*T3_K^4 + b*T3_K^3 + c*T3_K^2 + d*T3_K + e)
+    return EI_NOx
+end  # function EINOx4
+
+function EINOx(ac::aircraft, ip::Int; sp_humidity = 0.00634, method="cubic")
```

Comment:
This is the only place that uses `aircraft_type`. Would still vote against having it as a field and vote for having it as an input whenever `odperf!()` is called (which then passes to `EINOx` and co).

```suggestion
function EINOx(ac::aircraft, ip::Int, aircraft_type::Sym; sp_humidity = 0.00634, method="cubic")
```

Reply:
the motivation is that "narrow-body"/"wide-body" is too big of a user-facing lever that does nothing anywhere else. it's unintuitive

Reply:
perhaps a better var name is `correlation_type`, since that is the extent of its influence